File: retaguarda-software-house/php/retaguarda-sh/src/model/transiente/ObjetoPagSeguro.php
<?php

/// classe transiente que armazena o objeto enviado pelo PagSeguro
class ObjetoPagSeguro implements \JsonSerializable
{
	public $codigoTransacao;
	public $statusTransacao;
	public $codigoStatusTransacao;
	public $emailCliente;
	public $metodoPagamento;
	public $codigoTipoPagamento;
	public $codigoProduto;
	public $descricaoProduto;
	public $valorUnitario;

    /**
     * {@inheritdoc}
     */
    public function jsonSerialize()
    {
        return [
            'codigoTransacao' => $this->codigoTransacao,
            'statusTransacao' => $this->statusTransacao,
            'codigoStatusTransacao' => $this->codigoStatusTransacao,
            'emailCliente' => $this->emailCliente,
            'metodoPagamento' => $this->metodoPagamento,
            'codigoTipoPagamento' => $this->codigoTipoPagamento,
            'codigoProduto' => $this->codigoProduto,
            'descricaoProduto' => $this->descricaoProduto,
            'valorUnitario' => $this->valorUnitario
        ];
    }


}

File: retaguarda-software-house/c-sharp/T2TiRetaguardaSH/T2TiRetaguardaSH/Models/Transiente/ObjetoPagSeguro.cs
namespace T2TiRetaguardaSH.Models
{
    public class ObjetoPagSeguro
    {
        public ObjetoPagSeguro() { }

		public string CodigoTransacao { get; set; }
		public string StatusTransacao { get; set; }
		public string CodigoStatusTransacao { get; set; }
		public string EmailCliente { get; set; }
		public string MetodoPagamento { get; set; }
		public string CodigoTipoPagamento { get; set; }
		public string CodigoProduto { get; set; }
		public string DescricaoProduto { get; set; }
		public System.Nullable<System.Decimal> ValorUnitario { get; set; }
	}
}
